installing/installing_aws/installing-aws-default.adoc

```diff
@@ -29,7 +29,10 @@ program.
 ====
 * If you use a firewall, you must
 xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure it to allow the sites] that your cluster requires access to.
-
+* If you do not allow the system to manage identity and access management (IAM),
+then you can xref:../../installing/installing_aws/manually-creating-iam.adoc#manually-creating-iam[manually
```

Comment:
Suggest: "then a cluster administrator can [manually create and maintain IAM credentials]". I like clarifying it's not just creation but also on-going management across upgrades so folks know what they're getting into.